qiskit/quantum_info/operators/operator.py

```diff
@@ -572,6 +574,8 @@ def power(self, n: float, branch_cut_rotation=cmath.pi * 1e-12) -> Operator:
                 complex plane.  This shifts the branch cut away from the common point of :math:`-1`,
                 but can cause a different root to be selected as the principal root.  The rotation
                 is anticlockwise, following the standard convention for complex phase.
+            assume_unitary (bool): if `True`, the operator is assumed to be unitary. In this case,
+                for fractional powers we employ a faster implementation based on Schur's decomposition.
```

Comment:
```suggestion
            assume_unitary (bool): if ``True``, the operator is assumed to be unitary. In this case,
                for fractional powers we employ a faster implementation based on Schur's decomposition.
```

It might be worth mentioning that the output will just be wrong if this is set `True` and the matrix isn't unitary.

Reply:
A added a note, please see [47f7f90](https://github.com/Qiskit/qiskit/pull/13319/commits/47f7f907b51ad2404b2615e5058a9bd14cefef11).